src/test/resources/uk/gov/justice/hmpps/prison/api/resource/impl/offender_sentences.json

```diff
@@ -95,8 +95,7 @@
   },
   "dateOfBirth": "1970-03-01",
   "agencyLocationDesc": "LEEDS",
-  "internalLocationDesc": "A-1-5",
-  "facialImageId": -9
+  "internalLocationDesc": "A-1-5"
 }, {
```

Comment:
Necessary to remove the `facialImageId` check here - as this offender was also used for the image upload integration tests, and depending on the order they run in, may or may not be -9. Removing it makes this work in either order.